BrytescoreObjCAPI/BrytescoreObjCAPI/Classes/BrytescoreAPIManager.m

```diff
@@ -0,0 +1,787 @@
+//
+//  BrytescoreAPIManager.m
+//  BrytescoreObjcAPI
+//
+//  Created by Marisa Gomez on 10/16/18.
+//
+
+#import "BrytescoreAPIManager.h"
+
+//@interface BrytescoreAPIManager()
+//
+//- (void) track:(NSString*)eventName withEventDiplayName:(NSString*)eventDisplayName andData:(NSMutableDictionary*)data;
+//- (void) sendRequest:(NSString*)path withEventName:(NSString*)eventName eventDisplayName:(NSString*)eventDisplayName andData:(NSMutableDictionary*)data;
+//- (NSString*) generateUUID;
+//- (void) changeLoggedInUser:(NSInteger)userID;
+//- (void) heartBeat;
+//- (Boolean) updateUser:(NSMutableDictionary*)data;
+//- (Boolean) checkImpersonation:(NSMutableDictionary*)data;
+//- (void) checkHeartbeat;
+//- (void) print:(NSString*)item;
+//
+//@end
+
+@implementation BrytescoreAPIManager
+// --------------------------------- MARK: static variables --------------------------------- //
+// Variables used to fill event data for tracking
+static NSString* url = @"https://api.brytecore.com/";
+static NSString* packageUrl = @"https://cdn.brytecore.com/packages/";
+static NSString* packageName = @"/package.json";
+static NSString* hostname = @"com.brytecore.mobile";
+static NSString* library = @"iOS";
+static NSString* libraryVersion = @"0.0.0";
```

Comment:
Should we also be updating this to match the version when we deploy? I can't remember if we do that in the other versions, but we probably need to

Reply:
Yeah, we didn't do that in the other version. But I'll make that change so next push will include it